README.md

```diff
@@ -1,6 +1,192 @@
 # Shrinkwrap
 
-ShrinkWrap is a library to create archives from Java, which powers the [Arquillian](https://github.com/arquillian)
-deployment mechanism. Arquillian in turn is a library.
+ShrinkWrap is a Java library to create Java archives such as JARs, WARs, EARs and RARs.
+This is especially useful for integration testing, where you need to deploy specific configurations of your applications, such as [Arquillian](https://github.com/arquillian).
+
+## Requirements
+
+- **JDK**: Version 8 or newer.
+- **Maven**: Version 2.2.0 or newer.
+
+## Using Shrinkwrap
+
+To include ShrinkWrap in your project, you can add a dependency on the `shrinkwrap-depchain` module in your Maven `pom.xml`:
+
+```xml
+  <dependency>
+    <groupId>org.jboss.shrinkwrap</groupId>
+    <artifactId>shrinkwrap-depchain</artifactId>
+    <version>${version.shrinkwrap}</version>
+    <type>pom</type>
+  </dependency>
+```
+
+The `ShrinkWrap` class is the entry point to the ShrinkWrap library, enabling users to create and manage archives and domains.
+It provides static methods to create archives and domains with either default or custom configurations.
+
+### Archives
+
+`Archive` in ShrinkWrap represents a collection of resources, similar to a virtual filesystem.
+Each archive supports adding `Node` under `ArchivePath` , where `Node` can be directory or `Asset`.
+
+- `Node`: An entry in an `Archive`; may represent content or a directory
+- `ArchivePath`: Location in an `Archive` under which a `Node` lives
```

Comment:
Not sure about the "lives" part. Are there better alternatives? Exists? Is stored? Is Located? Resides? What do you think @xjusko

Reply:
Still valid @xjusko